tests/test_actors.py

```diff
@@ -59,6 +60,28 @@ def test_run(self):
         runner.run()
         self.assertEqual(self.count, len(MockDB.TASKS))
 
+    @patch('test_mock.MockDB.save')
+    def test_run_resourceconflict(self, mock_save):
+        """
+        Test the _run function, in case the DB throws a ResourceConflict
+        (when document exists with different revision or was deleted)
+        the _run function should continue
+        """
+        mock_save.side_effect = ResourceConflict
+        runner = ExampleRun(self._callback)
+        runner._run(task=Task({'_id': 'c', 'lock': None, 'done': None}), timeout=None)
+        self.assertEqual(runner.tasks_processed, 1)
+
+    @patch('test_mock.MockDB.save')
+    def test_run_exception(self, mock_save):
```

Comment:
pytest.raises as context_manager tests if an exception has been raised: https://docs.pytest.org/en/7.1.x/how-to/assert.html
You can easily test this, for example with:
```
        with pytest.raises(ValueError):
            mock_save.side_effect = Exception
```
This will cause the unit test to report an error, because the raised exception (Exception) is not the same as it expects  (ValueError).